lib/power_action_utils.pm

```diff
@@ -154,6 +154,12 @@ sub poweroff_x11 {
         send_key "alt-d";              # shut_d_own
         assert_screen 'logout-confirm-dialog', 10;
         send_key "alt-o";              # _o_k
+
+        if (check_screen('shutdown-auth')) {
+            record_soft_failure 'bsc#1076817 manually shutting down';
```

Comment:
the bug is specific about suspend and has received no update since 2018-02. Maybe you can create a new, specific bug report or at least write a comment in the bug?

Reply:
Added a comment to the bug